README.md

```diff
@@ -53,10 +53,11 @@ To run unit tests:
     python -m unittest discover -s tests
 
 ## Running in docker
-Ucs-service in docker shouldn't run in host mode, because rabbitmq service is built in docker and it may conflict with RackHD's rabbitmq server.
+RabbitMQ is not included in docker, user should install RabbitMQ separately.
```

Comment:
Suggest you add description as how user should configure RabbitMQ in this case. For example, User can install RabbitMQ in the host, or User can run RabbitMQ docker with host mode as well.